README.md

```diff
@@ -1 +1,2 @@
-# tribute-page-self-review
\ No newline at end of file
+# tribute-page-self-review
+cambios de prueba
```

Comment:
Kindly add a more descriptive readme file

Reply:
I think it will be good if you can include emojis and gifs in your reviews! 🤖 They make the code review more human-friendly.